test/uk/gov/hmrc/alcoholdutyaccount/models/subscription/AdrSubscriptionSummarySpec.scala

```diff
@@ -16,118 +16,123 @@
 
 package uk.gov.hmrc.alcoholdutyaccount.models.subscription
 
+import cats.implicits._
 import org.scalatest.matchers.should.Matchers.convertToAnyShouldWrapper
 import uk.gov.hmrc.alcoholdutyaccount.base.SpecBase
 import uk.gov.hmrc.alcoholdutyaccount.models.hods
-import uk.gov.hmrc.alcoholdutyaccount.models.hods.{ApprovalType, SubscriptionSummary}
 import uk.gov.hmrc.alcoholdutyaccount.models.subscription.ApprovalStatus.{Approved, DeRegistered, Insolvent, Revoked, SmallCiderProducer}
 
 class AdrSubscriptionSummarySpec extends SpecBase {
+  "AdrSubscriptionSummary" - {
```

Comment:
is feature toggle on or off here?

Reply:
I think the test names could be changed to make it a bit clearer, also my intelliJ complains that the line length is too long for the tests in the foreach which would be remedied by using more tiers in free spec

Reply:
Do you have a suggestion?

Reply:
I would say perhaps extract the string interpolated stuff as vals to describe what it actually is, so that it can be useful when debugging. Also renaming the test so that it is clearly talking about the feature toggle. And also making sure that the line length is below 160 chars so that intelliJ doesn't complain about it